docs/ecosystem/community/zio-temporal.md

```diff
@@ -0,0 +1,151 @@
+---
+id: zio-temporal
+title: "ZIO Temporal"
+---
+
+[ZIO Temporal](https://zio-temporal.vhonta.dev/) is a ZIO library for Temporal, a microservice workflow orchestration platform.
+
+## Introduction
+
+ZIO Temporal is a ZIO library based on the Temporal Java-SDK. ZIO Temporal brings first-class Scala & ZIO support with additional compile-time checks that Java SDK lacks.
+
+[Temporal](https://temporal.io/) platform helps to eliminate complex error or retry logic, avoid callbacks, and ensure that every workflow you start, completes. Temporal delivers durable execution for your services and applications.
+
+## Installation
+
+In order to use this library, we need to add the following dependency:
+
+```scala
+libraryDependencies += "dev.vhonta" %% "zio-temporal-core" % "0.2.0-M3"
+```
+
+## Example
+
+Here we have one ZIO app with two "modules", one which is the worker that executes the workflow activity and the other is the Client which sends the request to the Temporal platform. To run the sample, it's required to download [Temporal cli](https://github.com/temporalio/cli) (recommended for development).
+
+
+Run the Temporal server on one shell:
+
+```sh
+temporal server start-dev --ip 0.0.0.0 --db-filename /tmp/temporal.db
+```
+
+Now we can run the sample application which is based on Temporal Java SDK docs using [scala-cli](https://scala-cli.virtuslab.org):
+
+```scala
```

Comment:
To maintain docs in long run, it is better to have type-safe code blocks. I appreciate it if you take more effort to use `mdoc` modifiers to make sure the code compiles without error. Otherwise, it looks good to me.